### ISSUE TITLE
Why is the client able to access other clients' interaction data?

### ISSUE THREAD

**Issue by Miopha** [sichunluo/PerFedRec-Plus#3]
Thank you very much for your work.
In the local training code of `PerFedRec_plus.py`, why is a client able to access another client's interaction data?
`ego_embeddings = torch.sparse.mm(self.sparse_norm_adj, ego_embeddings)`
`self.sparse_norm_adj` is the normalized Laplacian matrix, which means all interaction information is stored locally. This is should be considered privacy leaking in federated learning.
